internal/pkg/cli/app_init.go

```diff
@@ -83,9 +83,12 @@ func (opts *InitAppOpts) Validate() error {
 		}
 	}
 	if opts.DockerfilePath != "" {
-		if _, err := listDockerfiles(opts.fs, opts.DockerfilePath); err != nil {
+		if _, err := opts.fs.Stat(opts.DockerfilePath); err != nil {
 			return err
 		}
+		if !strings.HasSuffix(opts.DockerfilePath, "/Dockerfile") {
+			return fmt.Errorf("a valid Dockerfile path is required")
+		}
```

Comment:
Should we validate the suffix before going to the filesystem?

Reply:
Yeah thought about that question, but can't figure out which comes first will be better. Do you have any idea why we should validate suffix first?

Reply:
Ultimately don't think it's a big deal, but would prefer to do a value sanity check first. If the input value isn't something we support then there's no point in searching the file system for it I think.